packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -100,92 +109,93 @@ export class ImportWallet extends React.Component<Props, State> {
     this.props.importBackupPhrase(formattedPhrase, false)
   }
 
-  onPressRestoreSocial = () => {
-    navigate(Screens.ImportWalletSocial)
+  shouldShowClipboard = (clipboardContent: string): boolean => {
+    return isValidBackupPhrase(clipboardContent)
   }
 
   render() {
-    const { backupPhrase } = this.state
-    const { t, isImportingWallet } = this.props
+    const { backupPhrase, keyboardVisible } = this.state
+    const { t, isImportingWallet, connected } = this.props
 
+    let codeStatus = CodeInputStatus.INPUTTING
+    if (isImportingWallet) {
+      codeStatus = CodeInputStatus.PROCESSING
+    }
     return (
-      <SafeAreaView style={styles.container}>
-        <KeyboardAwareScrollView
-          contentContainerStyle={styles.scrollContainer}
-          keyboardShouldPersistTaps="always"
-        >
-          <BackupKeyIcon style={styles.logo} width={140} height={102} />
-          <Text style={fontStyles.h1}>{t('title')}</Text>
-          <Text style={fontStyles.body}>{t('userYourBackupKey')}</Text>
-          <BackupPhraseContainer
-            onChangeText={this.setBackupPhrase}
-            value={backupPhrase}
-            testID="ImportWalletBackupKeyInputField"
-            mode={BackupPhraseContainerMode.INPUT}
-            type={BackupPhraseType.BACKUP_KEY}
-            style={componentStyles.marginTop20}
-          />
-          <Text style={styles.tip}>
-            <Text style={fontStyles.semiBold}>{t('tip')}</Text>
-            {t('backupKeyTip')}
-          </Text>
-        </KeyboardAwareScrollView>
-
-        {isImportingWallet && (
-          <View style={styles.loadingSpinnerContainer} testID="ImportWalletLoadingCircle">
-            <ActivityIndicator size="large" color={colors.celoGreen} />
-          </View>
+      <HeaderHeightContext.Consumer>
```

Comment:
Will other onboarding screen need these wrappers HeaderHeight and SafeArea wrappers? Wondering if we should just make a single wrapper component?

Reply:
Well, tbh I've just copied from what @jeanregisser has implemented. I guess we could created a wrapper out of it if need it more. @jeanregisser wdyt?